cdap-docs/admin-manual/source/operations/logging.rst

```diff
@@ -50,28 +121,234 @@ CDAP supports logging through standard `SLF4J (Simple Logging Facade for Java)
     output.emit(wordCount);
   }
 
-The log messages emitted by your application code can be viewed in different ways.
+The log messages emitted by your application code can be retrieved by:
 
 - Using the :ref:`restful-api`. The :ref:`Logging HTTP RESTful API <http-restful-api-logging>` 
   details the available contexts that can be called to retrieve different messages.
-- Log messages of an application can be viewed in the :ref:`CDAP UI <cdap-ui>`
-  by clicking the *Logs* tab of an application.
+- Log messages of a program can be viewed in the :ref:`CDAP UI <cdap-ui>`.
+  
+Application logs are stored in locations specified by properties in the
+:ref:`cdap-site.xml <appendix-cdap-default-logging>` file:
+
+- For **Standalone CDAP:** the property ``log.collection.root`` (default
+  ``${local.data.dir}/logs``) is the root location for collecting logs when in Standalone
+  CDAP.
+
+- For **Distributed CDAP:** the property ``log.base.dir`` (default ``/logs/avro``) is the
+  base directory in HDFS for logs
+  
+
+Configuring Logging
+===================
+Properties and configurations that control logging are located in Logback XML files and in
+the :ref:`cdap-site.xml <appendix-cdap-site.xml>` file.
 
+In the default configuration, CDAP uses Logback XML files located:
+
+- For Standalone CDAP: ``<cdap-sdk-home>/conf/logback.xml``
+- For Distributed CDAP: ``/opt/cdap/master/ext/logging/config``, as set by the property
+  ``log.process.pipeline.config.dir`` in the ``cdap-default.xml`` file
```

Comment:
these are for adding new logging pipelines. not relevant to configuring logging.